packages/@icon-magic/svg-to-png/src/index.ts

```diff
@@ -37,3 +38,37 @@ export async function convert(contents: string, options: SVGToPNGOptions): Promi
     });
   });
 }
+
```

Comment:
I'm wondering if we need this entire package at all. We could just have svg-to-img be a dependency of generate. Let's figure this out after we run the test the output sizes.

Reply:
yep I agree, it would just be one file with ~ 25 lines since we're removing the puppeteer stuff.